src/components/SingleArticle.jsx

```diff
@@ -55,7 +69,30 @@ const SingleArticle = ({ setVotes }) => {
         <footer>
```

Comment:
This isn't a footer, I think a `section` tag would be better here. A footer is rendered at the bottom of every page.